charts/platform-tenant/values.yaml

```diff
@@ -4,9 +4,10 @@
 
 nameOverride: ""
 fullnameOverride: ""
-# Formatted name of tenant
+# title formatted name of tenant
 title: ""
-
+# crdCheck for installed CRDS
+crdCheck: true
```

Comment:
Unsure we need this?

Reply:
useful for debugging, consider this a QOL of option